tasks/connectors/contrast/contrast.rb

```diff
@@ -0,0 +1,425 @@
+require_relative "lib/client"
+require "time"
```

Comment:
i thought time was included with ruby, is the require explicitly needed?

Reply:
Confirmed, I removed this.